installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh

```diff
@@ -0,0 +1,159 @@
+JENKINS_URL=http://$1/ # localhost or jenkins elb url
+JOB_NAME="Gitlab-Trigger-Job"
+USERNAME=
+PASSWORD=
+
+JENKINS_CLI=./jenkins-cli.jar
+
+cat <<EOF | java -jar $JENKINS_CLI -s $JENKINS_URL -auth $USERNAME:$PASSWORD create-job $JOB_NAME
+<?xml version='1.0' encoding='UTF-8'?>
+<flow-definition plugin="workflow-job@2.12">
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+    <com.dabsquared.gitlabjenkins.connection.GitLabConnectionProperty plugin="gitlab-plugin@1.5.2">
+      <gitLabConnection>Jazz-Gitlab</gitLabConnection>
+    </com.dabsquared.gitlabjenkins.connection.GitLabConnectionProperty>
+    <org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
+      <triggers>
+        <com.dabsquared.gitlabjenkins.GitLabPushTrigger plugin="gitlab-plugin@1.5.2">
+          <spec></spec>
+          <triggerOnPush>true</triggerOnPush>
+          <triggerOnMergeRequest>true</triggerOnMergeRequest>
+          <triggerOnPipelineEvent>false</triggerOnPipelineEvent>
+          <triggerOnAcceptedMergeRequest>false</triggerOnAcceptedMergeRequest>
+          <triggerOnClosedMergeRequest>false</triggerOnClosedMergeRequest>
+          <triggerOpenMergeRequestOnPush>never</triggerOpenMergeRequestOnPush>
+          <triggerOnNoteRequest>true</triggerOnNoteRequest>
+          <noteRegex>Jenkins please retry a build</noteRegex>
+          <ciSkip>true</ciSkip>
+          <skipWorkInProgressMergeRequest>true</skipWorkInProgressMergeRequest>
+          <setBuildDescription>true</setBuildDescription>
+          <branchFilterType>All</branchFilterType>
+          <includeBranchesSpec></includeBranchesSpec>
+          <excludeBranchesSpec></excludeBranchesSpec>
+          <targetBranchRegex></targetBranchRegex>
+          <secretToken>replace</secretToken>
+        </com.dabsquared.gitlabjenkins.GitLabPushTrigger>
+      </triggers>
+    </org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
+  </properties>
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition" plugin="workflow-cps@2.36">
+   <script>#!groovy.
+
+node  {
```

Comment:
Do we know why this code is in installer? Is this required? if yes, we probably should create a Jenkins file in core and reference it.

Reply:
This is a comment for code from 44 - 154

Reply:
Updated the code to handle this as a Jenkinsfile. 
But this file was merged to branch v1.4 in core. Hence, I couldn't find it in current active 1.3.1. All the gitlab core changes are a part of v1.4 in core. Are these changes added to v1.3.1 in jazz-core as well? Can you please address this.